manifests/moco-nodes.pp

```diff
@@ -229,391 +188,6 @@
     include toplevel::server::log_aggregator
 }
 
-# Signing workers
-node /^signingworker-.*\.srv\.releng\..*\.mozilla\.com$/ {
-    $aspects       = [ 'maximum-security' ]
-    $only_user_ssh = true
-    include toplevel::server::signingworker
-}
-
-# Signing scriptworkers
-node /^signing-linux-\d*\.srv\.releng\..*\.mozilla\.com$/ {
```

Comment:
We still use this for xpi, but it's pinned, so maybe we'll just keep that puppet env forked off an older rev.

Reply:
yeah, or just deploy directly to the worker. it will be moving to GCP soon, right?

Reply:
+1